contrib/cirrus/runner.sh

```diff
@@ -24,18 +24,15 @@ function _run_ext_svc() {
 }
 
 function _run_smoke() {
-    make gofmt
+    # Quick checks which do not require anything to be compiled
+    make gofmt lint validate.completions swagger-check tests-included
```

Comment:
`make tests-included` should only be run on PRs. I realize there's a new `skip smoke` in `.cirrus.yml` but it might be helpful to include a loud comment here explaining why this is meaningful only in PRs and why it should be skipped on tags/branches.

Reply:
Do you mean, `make tests-included` should only be run ***in CI*** for PRs?